### PR TITLE
DOC: Get rid of question marks in getting started tutorials

### LINKED ISSUE #48227: DOC: wrong question marks in tutorial
At https://pandas.pydata.org/docs/getting_started/intro_tutorials/index.html, things like

`How to create plots in pandas `

are not questions, so should not have a question mark.

(If it was "How do I create plots in pandas?", then it is a question, of course.)

### PR DESCRIPTION
**Pull request by samrao1997**
- [x] closes #48227 (Replace xxxx with the Github issue number)
- [ ] [Tests added and passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#writing-tests) if fixing a bug or adding a new feature
- [x] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).
- [ ] Added [type annotations](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#type-hints) to new arguments/methods/functions.
- [ ] Added an entry in the latest `doc/source/whatsnew/vX.X.X.rst` file if fixing a bug or adding a new feature.

### CODE REVIEW

File: doc/source/getting_started/intro_tutorials/04_plotting.rst [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 {{ header }}
 
-How do I create plots in pandas?
+How to create plots in pandas
 ----------------------------------
```
Comment (mroeschke) [inline doc/source/getting_started/intro_tutorials/04_plotting.rst:6]: All the `-` will need to match the length of these altered titles.

**Review by MarcoGorelli — CHANGES_REQUESTED** [review]
The issue is about removing question marks when the heading isn't a question (e.g. "How to calculate summary statistics?"), heading that are already questions (e.g. "How do I create plots in pandas?") can stay as they are

**Comment by samrao1997** [comment]
@MarcoGorelli I adjusted that to remain as is [aka `How do I create plots?`]. 

However, I noticed that in the docs for some reason the [plots](https://pandas.pydata.org/docs/getting_started/intro_tutorials/04_plotting.html#) heading shows `How to` with a question mark and when I added the previous change with `How to` without question mark building the docs got rid of the question mark on that page.

Not sure how the code show `How do I` when the docs show the heading `How to`....

File: doc/source/getting_started/intro_tutorials/06_calculate_statistics.rst [hunk 2]
```diff
@@ -30,7 +30,7 @@
         </ul>
     </div>
 
-How to calculate summary statistics?
+How to calculate summary statistics
 ------------------------------------
```
Comment (MarcoGorelli) [inline doc/source/getting_started/intro_tutorials/06_calculate_statistics.rst:34]: you need to align the underscores (remove the last one)

**Review by MarcoGorelli — CHANGES_REQUESTED** [review]

**Comment by MarcoGorelli** [comment]
> Not sure how the code show `How do I` when the docs show the heading `How to`....

you probably just need to do `python make.py clean` first

**Comment by samrao1997** [comment]
@MarcoGorelli I have updated the dashes to be the length of the alter titles. Sorry about that.

**Review by MarcoGorelli — APPROVED** [review]
Thanks @samrao1997

**Review by mroeschke — APPROVED** [review]

**Comment by mroeschke** [comment]
Thanks @samrao1997

**Pull request merged**